### ISSUE TITLE
CallAutomation + Speech Recognition 500 error

### ISSUE THREAD

**Issue by Frankkkkk** [Azure/azure-sdk-for-python#31638]
- **Package Name**:  `azure_communication_callautomation`
- **Package Version**: `callautomation/release/beta2`
- **Operating System**: linux
- **Python Version**: 3.10

**Describe the bug**
When using `start_recognizing_media` on a CallConnection, the server replies with the following error:
```
[{'data': {'callConnectionId': '411fc800-f4b7-4dae-8db2-aaee8c605ad9',
           'correlationId': 'f1372588-81f4-4615-a9b0-d835582e33d0',
           'resultInformation': {'code': 500,
                                 'message': 'Unknown internal server error.',
                                 'subCode': 9999},
           'serverCallId': 'aHR0cHM6Ly9hcGkuZmxpZ2h0cHJveHkuc2t5cGUuY29tL2FwaS92Mi9jcC9jb252LXN3Y2UtMDYuY29udi5za3lwZS5jb20vY29udi9KNm5fLWVqbTZrZXRwRGpjRDdoZExBP2k9NDMmZT02MzgyNzEyMjUzOTgwNDM5Njk='},
  'datacontenttype': 'application/json',
  'id': '584971e3-514e-4714-a4a2-b2ab28425e68',
  'source': 'calling/callConnections/411fc800-f4b7-4dae-8db2-aaee8c605ad9',
  'specversion': '1.0',
  'subject': 'calling/callConnections/411fc800-f4b7-4dae-8db2-aaee8c605ad9',
  'time': '2023-08-14T15:31:17.5278929+00:00',
  'type': 'Microsoft.Communication.RecognizeFailed'}]
```

**To Reproduce**
1. Setup a call and start recognizing speech input:
```python
# Answer a call
        call_connection_properties = cac.answer_call(
            incoming_call_context, event_callback_uri,
            cognitive_services_endpoint="https://switzerlandnorth.api.cognitive.microsoft.com/sts/v1.0/issuetoken")

# recognize input
            self.call_connection.start_recognizing_media(input_type="speech",
                                                         target_participant=self.caller,
                                                         speech_language="en-us",
                                                         end_silence_timeout=2000,
                                                         play_prompt=wanna_call)
```
**Expected behavior**
Speech input recognition should work

ping @vivekmore-msft @minwoolee-msft 

Cheers and thanks !

**Comment by Frankkkkk** [comment]
TL;DR: the `cognitive_services_endpoint` is wrong. It needs to be the endpoints of an "Azure AI services multi-service account".
Thanks and cheers !
